catalog/glue/glue.go

```diff
@@ -65,6 +65,10 @@ const (
 	Endpoint        = "glue.endpoint"
 	MaxRetries      = "glue.max-retries"
 	RetryMode       = "glue.retry-mode"
+
+	icebergFieldIDKey       = "iceberg.field.id"
+	icebergFieldOptionalKey = "iceberg.field.optional"
+	icebergFieldCurrentKey  = "iceberg.field.current"
```

Comment:
I forgot about this one. Will it ever be `false`? 🤔 The current schema should be stored in Glue.

Reply:
Interesting: https://github.com/apache/iceberg/pull/3888/